src/app/admin/adminActions.ts

```diff
@@ -0,0 +1,49 @@
+"use server";
+
+import { createServerActionClient } from "@supabase/auth-helpers-nextjs";
+import { cookies } from "next/headers";
+import { FunctionsResponse } from "@supabase/functions-js/src/types";
+import { User } from "@supabase/gotrue-js";
+import { Datum } from "@/components/Tables/Table";
+
+const supabase = createServerActionClient({ cookies });
+
+export interface UserRow extends Datum {
+    id: string;
+    email: string;
+    userRole: "admin" | "caller";
```

Comment:
should be from `Database["public"]["Enums"]["gender"]` instead, in case we want to add more rows!